operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc

```diff
@@ -38,8 +38,8 @@ include::modules/osdk-run-deployment.adoc[leveloffset=+2]
 [id="osdk-bundle-deploy-olm_{context}"]
 === Bundling an Operator and deploying with Operator Lifecycle Manager
 
-include::modules/osdk-bundle-operator.adoc[leveloffset=+4]
-include::modules/osdk-deploy-olm.adoc[leveloffset=+4]
+include::modules/osdk-bundle-operator.adoc[leveloffset=+3]
```

Comment:
@adellape Can you confirm that these leveloffset adjustments are correct (here and below 2 files)?

Reply:
LGTM